react-components/src/components/RevealToolbar/LayersButton/LayersButton.test.tsx

```diff
@@ -0,0 +1,98 @@
+/*!
+ * Copyright 2025 Cognite AS
+ */
+import { render, screen } from '@testing-library/react';
+import { describe, expect, test, vi } from 'vitest';
+import type { PropsWithChildren, ReactElement } from 'react';
+import { LayersButton } from './LayersButton';
+import type { LayersButtonProps } from './LayersButton';
+import { LayersButtonContext, type LayersButtonDependencies } from './LayersButton.context';
+import {
+  createCadHandlerMock,
+  createPointCloudHandlerMock,
+  createImage360HandlerMock
+} from '../../../../tests/tests-utilities/fixtures/modelHandler';
+
+describe(LayersButton.name, () => {
+  const mockCadHandler = createCadHandlerMock();
+  const mockPointCloudHandler = createPointCloudHandlerMock();
+  const mockImage360Handler = createImage360HandlerMock();
+  const defaultProps: LayersButtonProps = {
+    layersState: {
+      cadLayers: [{ revisionId: 456, applied: true, index: 0 }],
+      pointCloudLayers: [{ revisionId: 123, applied: true, index: 0 }],
+      image360Layers: [{ siteId: 'site-id', applied: true }]
+    },
+    setLayersState: vi.fn(),
+    defaultLayerConfiguration: undefined
+  };
+
+  const defaultDependencies: LayersButtonDependencies = {
+    useModelHandlers: vi.fn(() => [
+      {
+        cadHandlers: [mockCadHandler],
+        pointCloudHandlers: [mockPointCloudHandler],
+        image360Handlers: [mockImage360Handler]
+      },
+      vi.fn()
+    ]),
+    useSyncExternalLayersState: vi.fn(),
+    ModelLayerSelection: vi.fn(({ label }) => <div>{label}</div>)
+  };
+
+  test('renders without crashing', () => {
```

Comment:
Any basic thing we can assert here? F.ex. the existence of a button in the dom or something?